ui/src/app/utils/workbench-gapi-client.ts

```diff
@@ -0,0 +1,13 @@
+declare const gapi: any;
+
+export async function getBillingAccountName(workspaceNamespace: string) {
```

Comment:
This seems slightly overfit to me. I'd probably have it return the full JSON response structure, then have the client pick out `billingAccountName`